src/options/options.html

```diff
@@ -195,6 +189,13 @@ <h1 data-i18n="__MSG_titleAddonBehavior__">Addon behavior</h1>
 				<hr /><br />
 				<button data-i18n="__MSG_optionsResetButton__" type="button" id="resetButton">Reset all settings to defaults</button>
 			</div>
+			<div id="messageSuccess" aria-label="success message" data-i18n data-i18n-aria-label="__MSG_ariaMessageSuccess__" class="message-box success invisible fade-hide success-undo-message-box">
```

Comment:
Hmm, good idea in theory, but moving all success messages there is likely not a good idea.

The thing is, you actually do not want to show _all_ success messages here, because other modules could show other success messages and they should not be at the bottom. And if you just move the HTML element with id `#messageSuccess` that is just the [the default container for success messages](https://github.com/TinyWebEx/MessageHandler#parameters), but good news: the `MessageHandler` module can have more such arbitrary message locations/HTML elements. 
That is why I've opened https://github.com/TinyWebEx/AutomaticSettings/issues/28 and one would likely have to adjust that as well and then use the library here. (You can do so, it is a git submodule, online guides should tell you how to work with them…) See there for more details.

Reply:
So basically, I have to place the HTML I changed to the previous location, create a new id which displays the undo message, and then place a new HTML that only displays the undo message at the bottom. Did I get it right?

Reply:
Yeah that's it! The more or less tricky part is to then the fact that likely the AutomaticSettings library (module) needs to be adjusted to work with that message box instead of the success one - of course keep in mind it should at best have no breaking changes for other add-ons using the library, so that should be optional.

https://github.com/TinyWebEx/PermissionRequest does something similar/is used in the same way, as it also can register for arbitrary message boxes.